OsmAnd/src/net/osmand/plus/plugins/externalsensors/ExternalSensorsPlugin.java

```diff
@@ -72,7 +73,8 @@ public class ExternalSensorsPlugin extends OsmandPlugin {
 	public final CommonPreference<String> POWER_SENSOR_WRITE_TO_TRACK_DEVICE_ID;
 	public final CommonPreference<String> HEART_RATE_SENSOR_WRITE_TO_TRACK_DEVICE_ID;
 	public final CommonPreference<String> TEMPERATURE_SENSOR_WRITE_TO_TRACK_DEVICE_ID;
-	public final static String DENY_WRITE_SENSOR_DATA_TO_TRACK_KEY = "deny_write_sensor_data";
+	public final static String ANY_CONNECTED_DEVICE_WRITE_SENSOR_DATA_TO_TRACK_KEY = "any_connected_device_write_sensor_data_to_track_key";
```

Comment:
Will there be any issues with this change if we save it as a key in preference? Is migration needed?

Reply:
migration should not be needed. there should be checks if we can find device with such deviceId